.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 full.css
 .DS_Store
-.vscode
\ No newline at end of file
+.vscode
+.env
```

Comment:
why add the .env to git ignore? it is dynamic an subject to changes that matters to the entire team